backend/main/server/controllers.go

```diff
@@ -54,7 +54,7 @@ func (a *App) getResultsForProposal(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if *proposal.Computed_status == "closed" {
-		models.AddWinningVoteAchievement(a.DB, votes, results, proposal.Community_id)
+		models.AddWinningVoteAchievement(a.DB, votes, results)
```

Comment:
as we discussed, flag the proposal after all the vote stats have been updated so it doesnt attempt it more than once.